Userland/Libraries/LibELF/DynamicLinker.cpp

```diff
@@ -686,8 +686,12 @@ void ELF::DynamicLinker::linker_main(DeprecatedString&& main_program_path, int m
     if (s_do_breakpoint_trap_before_entry) {
 #if ARCH(AARCH64)
         asm("brk #0");
-#else
+#elif ARCH(RISCV64)
+        asm("ebreak");
```

Comment:
This might need a FIXME

Reply:
Ah ok, I interpreted it from its use in `longjmp` as something stronger than a break point

Reply:
Maybe use an intentionally illegal instruction, for them then, a quick google tells me the pseudo instruction`unimp` might do the job